product/.gitignore

```diff
@@ -1 +1,3 @@
-node_modules
\ No newline at end of file
+node_modules
+.env
+.env.test
```

Comment:
I think you should keep the .env.test in git (but without the actual data, just for example)